blockchain/projects.go

```diff
@@ -369,3 +350,54 @@ func (p *Projects) filterMissingExecutions(
 
 	return executions, nil
 }
+
+// GetAccount by the address along with its storage information.
+func (p *Projects) GetAccount(projectID uuid.UUID, address model.Address) (*model.Account, error) {
+	p.mutex.load(projectID).Lock()
```

Comment:
This should be read lock only -> `RLock()`

Reply:
Done